config/services.yml

```diff
@@ -851,6 +851,31 @@ https://cmr.uat.earthdata.nasa.gov:
         - image/tiff
       reprojection: true
 
+  - name: net2cog
```

Comment:
should this be 'podaac/net2cog'?

Reply:
Not sure, is the `podaac/` part significant?

Reply:
The only places this gets exposed I believe is in our documentation at harmony.earthdata.nasa.gov/docs 
![image](https://github.com/user-attachments/assets/811d00f2-589f-4938-a3e0-05db6e91442a) and our /versions endpoint. It's up to you if you want to include podaac there or not - it is not needed.

Reply:
I'm fine leaving it off